mithril-common/src/signable_builder/signable_builder_service.rs

```diff
@@ -65,7 +68,11 @@ impl SignableBuilderService for MithrilSignableBuilderService {
                 .with_context(|| format!(
                     "Signable builder service can not compute protocol message with beacon: '{beacon}'"
                 ))?,
-            SignedEntityType::CardanoStakeDistribution(_) => todo!(),
+            SignedEntityType::CardanoStakeDistribution(e) => self
+                .cardano_stake_distribution_builder
+                .compute_protocol_message(e)
+                .await
+                .with_context(|| "Signable builder service can not compute protocol message with epoch: '{e}")?,
```

Comment:
Maybe the error contexts should also mention the signed entity type as this message is the same as in `MithrilStakeDistribution`

Reply:
Done ✅